modules/doc/content/newsletter/2022_02.md

```diff
@@ -1,9 +1,13 @@
 # MOOSE Newsletter (February 2022)
 
-!alert! construction title=In Progress
-This MOOSE Newsletter edition is in progress. Please check back in March 2022
-for a complete description of all MOOSE changes.
-!alert-end!
+## Legacy Input Parameter Removal
+
+As announced in November 2021, the legacy input parameter construction method
+(`InputParameters validParams<T>()`) is being removed on 1 March 2022 after two
+years of support. Any remaining templated methods will not compile after this date.
+For information on how to convert source code to the current input parameter
+construction method, directions can be found in the
+[original deprecation announcement](2021_11.md#legacy-input-param-deprecation).
```

Comment:
@loganharbour I added a legacy input parameter removal notice - if you think we need something more expansive, let me know. I added a link to the fixup directions for those who still need it.

Reply:
Nope, this is perfect 👍🏼